care/facility/api/serializers/prescription.py

```diff
@@ -75,7 +75,21 @@ def validate(self, attrs):
                     }
                 )
 
-        if attrs.get("is_prn"):
+        if attrs.get("dosage_type") == "TITRATED":
+            if not attrs.get("base_dosage"):
+                raise serializers.ValidationError(
+                    {
+                        "base_dosage": "Base dosage should be set for titrated prescriptions."
+                    }
+                )
```

Comment:
`base_dosage` is mandatory for all types of prescriptions right? Not just for titrated right?

Reply:
Also, we should add required validation for max_dosage if it is PRN prescription right?